processor/error/event.go

```diff
@@ -90,6 +90,15 @@ func (e *Event) Transform(config *pr.Config, service m.Service) common.MapStr {
 	return e.data
 }
 
+// This updates the event in place
+func (e *Event) contextTransform(pa *payload) common.MapStr {
+	if e.Context == nil {
+		e.Context = make(map[string]interface{})
+	}
+	utility.InsertInMap(e.Context, "user", pa.User)
+	return e.Context
+}
```

Comment:
I suggest to create a `model.Context` which still can be a `common.MapStr`, but to be consistent with the other attributes, also has its own `transform` method, which then can add `user`, `system`, ..